src/nft/components/collection/Activity.tsx

```diff
@@ -98,19 +98,20 @@ export const Activity = ({ contractAddress, rarityVerified, collectionName, chai
   const Filter = useCallback(
     function ActivityFilter({ eventType }: { eventType: ActivityEventType }) {
       const isActive = activeFilters[eventType]
-      const activeBackgroundColor = isDarkMode ? vars.color.gray500 : vars.color.gray200
 
       return (
         <FilterBox
           className={styles.filter}
-          backgroundColor={isActive ? activeBackgroundColor : themeVars.colors.backgroundInteractive}
+          backgroundColor={isActive ? themeVars.colors.surface3 : themeVars.colors.surface1}
+          color={isActive ? themeVars.colors.neutral1 : themeVars.colors.neutral1}
+          border={`1px solid ${isActive ? themeVars.colors.surface3 : themeVars.colors.surface3}`}
```

Comment:
nit: with all these settings you could just pass `isActive` to the component and then determine the styling in the component